### PR TITLE
 Add PEER_ADDRESS tag

### PR DESCRIPTION
**Pull request by qudongfang**
Add `peer.address` tag according to [OpenTracing Specification Semantic Conventions v1.1](https://github.com/opentracing/specification/blob/master/semantic_conventions.md).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,16 +2,15 @@
 
 If you would like to contribute code you can do so through GitHub by forking the repository and sending a pull request (on a branch other than `master` or `gh-pages`).
 
-
 ## License
 
-By contributing your code, you agree to license your contribution under the terms of the APLv2: https://github.com/opentracing/opentracing-java/blob/master/LICENSE
+By contributing your code, you agree to license your contribution under the terms of the APLv2: <https://github.com/opentracing/opentracing-java/blob/master/LICENSE>
```
Comment (yurishkuro) [inline CONTRIBUTING.md]: changes to this file don't seem to be needed

Comment (qudongfang) [reply CONTRIBUTING.md]: Uh,  maybe you are right.
This is a markdown syntax warning that bothers me.
I can revert this file if you insist.

Comment (yurishkuro) [reply CONTRIBUTING.md]: it's quite normal to have bare links in .md. Brackets just make it harder to copy if someone needs to.

Comment (qudongfang) [reply CONTRIBUTING.md]: Good point. I'll revert this file then.
Do you need me to rebase these two commits to one?

**Comment by yurishkuro** [comment]
@carlosalberto can you approve / merge?

**Pull request merged**